replate/src/context/types.js

```diff
@@ -0,0 +1,32 @@
+// variables that represent strings that allow me to change my state within my reducer function.
+
```

Comment:
@codenamerockey So you're using useReducer as well in conjunction with useContext?  just clarifying